pentaho-js/src/main/javascript/cdf/components/AnalyzerComponent.js

```diff
@@ -43,19 +44,27 @@ define([
     },
 
     getOptions: function() {
+      var myself = this;
       var options = {
         solution: this.solution,
         path: this.path,
         action: this.action,
         command: this.command == undefined ? "open" : this.command,
         showFieldList: this.showFieldList == undefined ? false : this.showFieldList,
         showRepositoryButtons: this.showRepositoryButtons == undefined ? false : this.showRepositoryButtons,
-        frameless: this.frameless == undefined ? false : this.frameless
+        frameless: this.frameless == undefined ? false : this.frameless,
+        dateFormats: this.dateFormats == undefined ? {} : this.dateFormats
```

Comment:
Nice catch, these are the changes needed https://github.com/afrjorge/cdf/commit/fd20c0b45e716d561deb1ee90b02165db8464f94